### ISSUE TITLE
Azure.FrontDoorWAF.Exclusions reports exclusions when none is specified

### ISSUE THREAD

**Issue by BernieWhite** [Azure/PSRule.Rules.Azure#1751]
**Description of the issue**

If the `exclusions` property of a `managedRuleSets` does not exist, `Azure.FrontDoorWAF.Exclusions` returns a false positive.

Also applies to `Azure.AppGwWAF.Exclusions`.

**Expected behaviour**

The rule should pass if `exclusions` is not set or empty.

**Error output**

```text
Path Properties.managedRules.managedRuleSets[0].exclusions: The field 'Properties.managedRules.managedRuleSets[0].exclusions' does not exist.
```

**Module in use and version:**

- Module: PSRule.Rules.Azure
- Version: **1.20.0-B0223**
